recipes/configure.rb

```diff
@@ -19,6 +19,8 @@
 #
 
 include_recipe 'apt'
+include_recipe 'yum'
```

Comment:
yum itself probably isn't necessary. That doesn't prime cache like apt. It just configures the main yum config file